geocoding.go

```diff
@@ -40,6 +46,38 @@ type ForwardGeocodeRequest struct {
 	Types        Types
 }
 
+func (r ForwardGeocodeRequest) MarshalJSON() ([]byte, error) {
+	type forwardGeocodeRequest struct {
```

Comment:
nit: It's generally a good practice to group similar fields(eg: strings together, bool together, slices together: "like birds of same feather flock together" :-)  for better readability.